packages/builder/src/package.ts

```diff
@@ -137,7 +137,8 @@ export async function copyPackage({ packageRef, tags, variant, fromStorage, toSt
     const preCtx = await createInitialContext(def, deployInfo.meta, 0, deployInfo.options);
 
     return {
-      packagesNames: [def.getVersion(preCtx), ...(context ? context.tags || [] : tags)].map(
+      // TODO (FIX): When using an interpolated <%= package.version %>, def.getVersion doesnt return a value properly.
+      packagesNames: [def.getVersion(preCtx) || version, ...(context ? context.tags || [] : tags)].map(
```

Comment:
When publishing a package that has the interpolated package value like `<%= package.version %>`, `def.getVersion` fails to parse out the value from it and I'm not sure what's causing it. @mjlescano If you dont mind taking a look at this that would be great.